lib/beaker/host/windows.rb

```diff
@@ -39,9 +39,10 @@ def self.foss_defaults
         'user'              => 'Administrator',
         'group'             => 'Administrators',
         'puppetpath'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
-        'puppetconfdir'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
-        'puppetcodedir'        => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
+        'puppetconfdir'     => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
+        'puppetcodedir'     => '`cygpath -smF 35`/PuppetLabs/puppet/etc',
         'hieraconf'         => '`cygpath -smF 35`/Puppetlabs/puppet/etc/hiera.yaml',
+        'hieradatadir'      => '`cygpath -smF 35`/PuppetLabs/puppet/etc/hieradata',
```

Comment:
@kevpl I would leave out the pathing changes for now, because we have not finalized them. Currently the plan calls for Windows `codedir` to be `$confdir/code`, which is unlike Unix. We are probably going to change Windows to use `.../PuppetLabs/puppet/code` instead. And if that happens, then `hieraconf`, `hieradatadir`, `distmoduledir` all need to move under `$codedir`.